README.md

```diff
@@ -1142,7 +1144,8 @@ Other Style Guides
     const luke = new Jedi();
 
     luke.jump()
-      .setHeight(20);
+      .setHeight(20); // same as var tmp = luke.jump(); tmp.setHeight(20);
+    // executes left to right. First jump() and then setHeight()
```

Comment:
is there any reason to believe this comment isn't self-evident?

Reply:
@ljharb I agree with you that line 1148 can be removed. Is that okay?